go-controller/pkg/cluster/cluster.go

```diff
@@ -72,6 +72,20 @@ func setupOVNNode(nodeName string) error {
 	if err != nil {
 		return fmt.Errorf("error setting OVS external IDs: %v\n  %q", err, stderr)
 	}
+	// If EncapPort is not the default tell sbdb to use specified port.
+	if config.Default.EncapPort != config.DefaultEncapPort {
+		uuID, stderr, errGet := util.RunOVSVsctl("--if-exists", "get", "Open_vSwitch", ".",
+			"external_ids:system-id")
+		if errGet != nil {
+			return fmt.Errorf("Failed to get system-id stderr:%s (%v)", stderr, err)
```

Comment:
Did you mean `errGet` instead of `err` in Errof() above?

Reply:
yes, good catch